client/driver/docker.go

```diff
@@ -408,6 +409,14 @@ func (d *DockerDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle
 	if err := mapstructure.WeakDecode(task.Config, &driverConfig); err != nil {
 		return nil, err
 	}
+
+	if strings.Contains(driverConfig.ImageName, "https://") {
```

Comment:
@ErikEvenson Oh I see. Can we create another method called ```Init()``` then? This could be called before Validate()